config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/WSDL.java

```diff
@@ -0,0 +1,35 @@
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class WSDL {
+
+    @JsonIgnore
+    private String rootUrl;
```

Comment:
I think we should not ignore the rootUrl, if it is the link to Wsdl. When building, getRootUrl will return nulls.

Reply:
yeah you are right.. I was having problem while running build already..